### PR TITLE
PXB-2682 missing download command in xbcloud documentation (2.4)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified: source/xbcloud/xbcloud.rst

### CODE REVIEW

**Review by rahulmalik87 — CHANGES_REQUESTED** [review]
Can you please fix the jira ticket in github and commit messages. It should be PXB-2682 and not PXB-2489.

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst [hunk 1]
```diff
@@ -367,10 +366,10 @@ Then you can make the incremental backup:
    --swift-auth-url=http://127.0.0.1:35357/ --parallel=10 \
    inc_backup
 
-Preparing incremental backups
+Preparing an incremental backups
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst]: backups -> backup

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst [hunk 2]
```diff
@@ -367,10 +366,10 @@ Then you can make the incremental backup:
...
 --------------------------------------------------------------------------------
 
-To prepare a backup you first need to download the full backup:
+To prepare a backup, you download the full backup:
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst]: you download -> download

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst [hunk 3]
```diff
@@ -423,8 +422,7 @@ you can specify only tables you want to restore:
 
    $ xbstream -xv -C /storage/partial < /storage/partial/partial.xbs
 
-This command will download just ``ibdata1`` and ``sakila/payment.ibd`` table
-from the full backup.
+This command downloads just the ``ibdata1`` table and the ``sakila/payment.ibd`` table from a full backup.
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst]: just the ``ibdata1`` table -> the ``ibdata1`` table

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst [hunk 4]
```diff
@@ -448,7 +446,7 @@ Command-line options
 
 .. option:: --swift-storage-url
 
-   xbcloud will try to get object-store URL for given region (if any specified)
+   xbcloud attempts to get object-store URL for given region (if any specified)
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst]: xbcloud attempts to get an object-store URL for a specified region.

**Comment by patrickbirch** [comment]
Fixed  jira ticket in github and commit messages.

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst]: Done

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst (hunk 3 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst]: Removed just

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst (hunk 4 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst]: Fixed.

**Review by alina-derkach-oaza — APPROVED** [review]

**Review by alina-derkach-oaza — APPROVED** [review]

**Review by rahulmalik87 — APPROVED** [review]
LGTM

**Pull request merged**